### PR TITLE
feat: add scalable prop to typography

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Add an optional scalable prop to typography that adds our density based functions (...applyScaledTypeRamp()) via className.

Closes: #1759 

## Motivation & context
https://github.com/microsoft/fast-dna/issues/1759

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

**Comment by janechu** [comment]
@chrisdholt let's get some opinion from @bheston

**Comment by chrisdholt** [comment]
@janechu - sure, I think simply from a systematic standpoint though I would assume if we want to opt certain regions out of density we would do that via context.

**Comment by eljefe223** [comment]
> I'm for this, but I do wonder if a prop is needed or if we should just scale the typography component by default.
> 
> Here's the thought I'm working through:
> 
> From a design system standpoint, density is something that you opt into all up. If you want parts of the UI to respond to density, but not others, we have our `<DesignSystemProvider />` component that can reset the context and density for given elements or parts of the UI.
> 
> I'm struggling to know why we don't just wire this component for density with that in mind. I either change the density, or I don't. If I want to exempt a portion of UI, I can reset the multiplier for that context.

@bheston How do you feel about this?

File: packages/fast-components-react-base/src/typography/typography.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ class Typography extends Foundation<
 > {
     public static defaultProps: Partial<TypographyProps> = {
         tag: TypographyTag.p,
+        scalable: false,
```
Comment (nicholasrice) [inline packages/fast-components-react-base/src/typography/typography.tsx]: I prefer the name "scaled" - like you used in the class-name. To me, "scalable" implies I can take some action to scale this - not that the property itself causes scaling.

**Comment by nicholasrice** [comment]
I second @chrisdholt - I'll also add the we're effectively doubling the size of this stylesheet for the sake of a single opt-in prop with this approach.

**Comment by eljefe223** [comment]
Closing pull request in favor of just implementing `...applyScaledTypeRamp()` by default.

New PR can be found here: https://github.com/microsoft/fast-dna/pull/1777

**Pull request closed without merging**